WorldsAdriftServer/Objects/SocialObjects/CharacterDataModel.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Newtonsoft.Json;
+using WorldsAdriftServer.Helpers;
```

Comment:
i think this line is not needed (and throws an error for me because of the last `s`)